manifests/d/denoland/deno/1.25.3/denoland.deno.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.1.6 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.4.0/manifest.installer.1.4.0.json
+
+PackageIdentifier: denoland.deno
```

Comment:
```suggestion
PackageIdentifier: DenoLand.Deno
```
😉

Reply:
I was going based on the github repo

Reply:
I perfer this one because they have a company called Deno Land Inc., and they usually type Deno as `Deno` instead of something else.

Reply:
I also prefer `DenoLand.Deno` because that's how it's described on their site.